### PR TITLE
Bump Story Promo and remove extended Link styles

### PR DESCRIPTION
**Pull request by DenisHdz**
Resolves #NUMBER

**Overall change:**
In [this](https://github.com/bbc/simorgh/pull/6248) PR we extended the Story Promo `Link` styles to apply the `overflow-wrap: break-word;` property to fix the long headlines overflowing the main content on Opera Mini Extreme Mode.

This property has been now applied directly in the Psammead component https://github.com/bbc/psammead/pull/3410, so we can bump the version of the package and remove the initial styles applied in Simorgh to avoid a duplication of the same property.

**Code changes:**
- Remove `Link` styles extension
- Bump Story Promo
- Update snapshots

---

- [X] I have assigned myself to this PR and the corresponding issues
- [X] I have added labels to this PR for the relevant pod(s) affected by these changes
- [X] I have assigned this PR to the Simorgh project

**Testing:**

- [X] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] If necessary, I have run the local E2E non-smoke tests relevant to my changes (`CYPRESS_APP_ENV=local CYPRESS_SMOKE=false npm run test:e2e:interactive`)
- [x] This PR requires manual testing

### CODE REVIEW

**Review by EinsteinNjoroge — APPROVED** [review]

File: package-lock.json [hunk 1]
```diff
@@ -8119,7 +8119,7 @@
     },
     "diffable-html": {
       "version": "4.0.0",
-      "resolved": "https://registry.npmjs.org/diffable-html/-/diffable-html-4.0.0.tgz",
+      "resolved": "https://npm.morph.int.tools.bbc.co.uk/diffable-html/-/diffable-html-4.0.0.tgz",
```
Comment (OlgaLyubin) [inline package-lock.json]: Looks good, just wondering if this change is needed along with the one below for `jest-serializer-html` ?

File: package-lock.json [hunk 2]
```diff
@@ -14059,7 +14059,7 @@
     },
     "jest-serializer-html": {
       "version": "7.0.0",
-      "resolved": "https://registry.npmjs.org/jest-serializer-html/-/jest-serializer-html-7.0.0.tgz",
+      "resolved": "https://npm.morph.int.tools.bbc.co.uk/jest-serializer-html/-/jest-serializer-html-7.0.0.tgz",
```
Comment (DenisHdz) [inline package-lock.json]:
```suggestion
      "resolved": "https://registry.npmjs.org/jest-serializer-html/-/jest-serializer-html-7.0.0.tgz",
```

File: package-lock.json (hunk 1 above)
Comment (DenisHdz) [inline package-lock.json]:
```suggestion
      "resolved": "https://registry.npmjs.org/diffable-html/-/diffable-html-4.0.0.tgz",
```

Comment (DenisHdz) [reply package-lock.json]: For some reason, those were updated after running `npm ci`. I've reverted them.

**Review by OlgaLyubin — APPROVED** [review]
Looks good

**Comment by PriyaKR** [comment]
LGTM.

**Pull request merged**